src/pykeen/nn/functional.py

```diff
@@ -1186,3 +1189,49 @@ def cross_e_interaction(
         x = dropout(x)
     # similarity
     return (x * t).sum(dim=-1)
+
+
+def boxe_interaction(
+    # head
+    h_pos: torch.FloatTensor,
+    h_bump: torch.FloatTensor,
+    # relation box: head
+    rh_base: torch.FloatTensor,
+    rh_delta: torch.FloatTensor,
+    rh_size: torch.FloatTensor,
+    # relation box: tail
+    rt_base: torch.FloatTensor,
+    rt_delta: torch.FloatTensor,
+    rt_size: torch.FloatTensor,
+    # tail
+    t_pos: torch.FloatTensor,
+    t_bump: torch.FloatTensor,
+    # power norm
+    tanh_map: bool = False,
+    p: int = 2,
+    power_norm: bool = False,
+) -> torch.FloatTensor:
+    """
+    This interaction relies on Abboud's point-to-box distance
+    :func:`pykeen.utils.point_to_box_distance`.
+    """
+    # TODO: Docstring
```

Comment:
TODO: docstring should very explicitly explain what each of the embeddings does / why